doc/source/user/how-to-how-to.rst

```diff
@@ -0,0 +1,114 @@
+.. _how-to-how-to:
+
+##############################################################################
+How to write a NumPy how-to
+##############################################################################
+
+How-tos get straight to the point -- they
+
+  - answer a focused question, or
+  - narrow a broad question into focused questions that the user can
+    choose among.
+
+******************************************************************************
+A stranger has asked for directions...
+******************************************************************************
+
+**"I need to fill my tank."**
+
+******************************************************************************
+Give a brief but explicit answer
+******************************************************************************
+
+  - `"Three miles, take a right at Hayseed Road, it's on your left."`

```

Comment:
Let's use the metric system, not miles. "Three kilometers, take a right at Hayseed Road, the petrol station is on your left."